rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -60,6 +60,25 @@ clean_cf_lock_files() {
   rm -f ${CFE_DIR}/state/cf_lock.${CFENGINE_DB_EXT}.lock
 }
 
+fix_perm() {
+  perm="$1"
+  file="$2"
+  if [ `stat -c %a "${file}"` != "${perm}" ]; then
```

Comment:
We could just do the chmod as before but the goal was to avoid doing it unnecessarily.
Do you think it is a worthy goal ?